doc-content/enterprise-only/processes/creating-new-users-proc.adoc

```diff
@@ -19,8 +19,7 @@ image::Designer/add-roles.png[Add roles menu location]
 . Select *admin* and *user* and click *Add to selected roles*.
 . Click *Create* to assign `Katy` to the *admin* and *user* roles.
 . Click *Yes* to set the password to `katy` and click *Change*.
-. Click the *Users* tab and click image:Designer/new-user.png[].
-. Click *New user*, enter `Bill`, and click *Next*.
+. Create another *New user*, enter `Bill`, and click *Next*.
 . Click the *Groups* tab and click image:Designer/add-groups.png[].
 . Select *rest-all* and *kie-server* and click *Add to selected groups*.
```

Comment:
`rest-all` group is not needed and instead of `kie-server` you could use `user` instead and that you have in roles so skip groups completely here I'd say